### PR TITLE
Updates for OpenFOAM containers and support

### PR DESCRIPTION
**Pull request by AlexisEspinosaGayosso**
Once again, here are the changes needed for OpenFOAM containers that Pawsey is currently supporting

### CODE REVIEW

File: quay.io/pawsey/openfoam-org/container.yaml [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-  2.2.0: aliases/2.2.0.yaml
-  2.4.x: aliases/2.4.x.yaml
-  5.x: aliases/5.x.yaml
-  '7': aliases/7.yaml
+  '10': aliases/10.yaml
+  '9': aliases/9.yaml
   '8': aliases/8.yaml
-#aliases:
+  '7': aliases/7.yaml
+  #5.x: aliases/5.x.yaml
```
[the remark this answers is not in the document]
Comment (AlexisEspinosaGayosso) [reply quay.io/pawsey/openfoam-org/container.yaml:20]: No. So I will then remove all that is not longer supported. No comments.

[the remark this answers is not in the document]
Comment (AlexisEspinosaGayosso) [reply quay.io/pawsey/openfoam-org/container.yaml]: Done. All lines and files related to unsupported versions have been removed.

**Comment by vsoch** [comment]
@marcodelapierre good for me when you give the final :+1: !

**Comment by marcodelapierre** [comment]
@AlexisEspinosaGayosso  thanks for this one. 

Assuming that the long lists of aliases for each version are accurate, it is good to go for me!

**Review by vsoch — APPROVED** [review]

**Pull request merged**